spot-ml/src/main/scala/org/apache/spot/dns/DNSWordCreation.scala

```diff
@@ -79,18 +81,23 @@ class DNSWordCreation(frameLengthCuts: Array[Double],
               dnsQueryType: Int,
               dnsQueryRcode: Int): String = {
 
+    Try {
+      val DomainInfo(domain, topDomain, subdomain, subdomainLength, subdomainEntropy, numPeriods) =
+        extractDomainInfo(queryName, topDomainsBC, userDomain)
 
-    val DomainInfo(domain, topDomain, subdomain, subdomainLength, subdomainEntropy, numPeriods) =
-      extractDomainInfo(queryName, topDomainsBC, userDomain)
+      Seq(topDomain,
+        Quantiles.bin(frameLength.toDouble, frameLengthCuts),
+        Quantiles.bin(unixTimeStamp.toDouble, timeCuts),
+        Quantiles.bin(subdomainLength.toDouble, subdomainLengthCuts),
+        Quantiles.bin(subdomainEntropy, entropyCuts),
+        Quantiles.bin(numPeriods.toDouble, numberPeriodsCuts),
+        dnsQueryType,
+        dnsQueryRcode).mkString("_")
+    } match {
+      case Success(word) => word
+      case _ => "word_error"
```

Comment:
yeah, we should be getting "word_error" from a named constant in some object or package